### PR TITLE
70 to collect all inputs

### PR DESCRIPTION
**Pull request by Kyslytsia**

### CODE REVIEW

File: client/src/components/checkbox-radio/index.tsx [hunk 1]
```diff
@@ -33,8 +36,10 @@ const CheckboxRadio = ({
     <label className={styles.label}>
       <div className={checkboxClass}>
         <input
+          ref={itemRef}
```
Comment (Wolshebnik) [inline client/src/components/checkbox-radio/index.tsx:39]: sort

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

File: client/src/app/[locale]/(admin)/dashboard/config.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  agree: boolean;
+  lastName: string;
+  middleName: string;
+  socialNetworks: string;
+  organizationName: string;
+  positionOrganization: string;
+  organizationTaxNumber: string;
+  certificateOfRegister: string;
+  dateOfRegisterOrganization: string;
+}
```
Comment (rpkyrych) [inline client/src/app/[locale]/(admin)/dashboard/config.ts:19]: It'd be better to use OOP paradigm here: spit the object into a few other (Organization, Representative) and to use aggregation for those objects.

File: client/src/app/[locale]/(admin)/dashboard/config.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  middleName: data?.middleName ?? '',
+  socialNetworks: data?.socialNetworks ?? '',
+  organizationName: data?.organizationName ?? '',
+  positionOrganization: data?.positionOrganization ?? '',
+  organizationTaxNumber: data?.organizationTaxNumber ?? '',
+  certificateOfRegister: data?.certificateOfRegister ?? '',
+  dateOfRegisterOrganization: data?.dateOfRegisterOrganization ?? '',
+});
+
+export const validationSchema = (
```
Comment (rpkyrych) [inline client/src/app/[locale]/(admin)/dashboard/config.ts:37]: The name of the constant doesn't provide an information to whom those validation belong. It'd be more beneficial to rename it.

File: client/src/app/[locale]/(admin)/dashboard/page.tsx [hunk 4]
```diff
@@ -1,5 +1,273 @@
...
+  const [isOpenAccept, setIsOpenAccept] = useState<boolean>(false);
+  const [isOpenDecline, setIsOpenDecline] = useState<boolean>(false);
+  const router = useRouter();
+
+  const error = useTranslations('validation');
+  const text = useTranslations('auth-page.registration');
+  const dashboard = useTranslations('auth-page.dashboard');
+
+  const onSubmit = (values: FormikValues) => {
+    console.log('data', values);
```
Comment (rpkyrych) [inline client/src/app/[locale]/(admin)/dashboard/page.tsx:33]: The log message should better describe logged value

File: client/src/components/checkbox-radio-field/index.tsx [hunk 5]
```diff
@@ -9,17 +9,21 @@ import { ICheckboxProps } from '../checkbox-radio/types';
...
     <Field name={name}>
-      {({ meta, form, field: { ...fieldProps } }: FieldProps) => {
+      {({ meta, form, field: { value, ...fieldProps } }: FieldProps) => {
         const error = Error.controlError(meta, name, label);
+        const checked = type === 'radio' ? value === label : value;
 
         const change = async (e: ChangeEvent<HTMLInputElement>) => {
           if (name) {
-            await form.setFieldValue(name, e.target.checked);
+            const value = type === 'radio' ? e.target.value : e.target.checked;
```
Comment (rpkyrych) [inline client/src/components/checkbox-radio-field/index.tsx:24]: The name of the variable doesn't explain what is the content or purpose of the value in it.

File: client/src/app/[locale]/(admin)/dashboard/[id]/page.tsx [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+  Accordion,
+  FileField,
+  InputField,
+  ButtonIcon,
+  ModalAdmin,
+} from '@/components';
+
+import { ModalContent } from './components/modal-content';
+
+const Dashboard1 = () => {
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(admin)/dashboard/[id]/page.tsx:24]: so-so Name

File: client/src/app/[locale]/(admin)/dashboard/[id]/page.tsx [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+  const [inputFields, setInputFields] = useState<string[]>(['socialNetworks']);
+  const [isOpenSave, setIsOpenSave] = useState<boolean>(false);
+  const [isOpenAccept, setIsOpenAccept] = useState<boolean>(false);
+  const [isOpenDecline, setIsOpenDecline] = useState<boolean>(false);
+  const router = useRouter();
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(admin)/dashboard/[id]/page.tsx:29]: should be moved to the top of the list

File: client/src/app/[locale]/(login)/sign-up/sign-up.tsx [hunk 8]
```diff
@@ -4,14 +4,21 @@ import Link from 'next/link';
...
 import { Form, Formik, FormikValues } from 'formik';
 
-import { Title } from '@/components/title';
-import { CheckboxRadioField } from '@/components/checkbox-radio-field';
-import { Button, DateField, FileField, InputField } from '@/components';
-
-import { initialValues, validationSchema } from './config';
+import {
+  organizationInitialValues,
+  organizationValidation,
```
Comment (Wolshebnik) [inline client/src/app/[locale]/(login)/sign-up/sign-up.tsx:9]: sort

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Review by rpkyrych — APPROVED** [review]

**Pull request merged**